content/learning-paths/smartphones-and-mobile/afrc/how-to-6.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Verify that compression is applied
+weight: 7
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Verify that compression is applied
+
+To query which compression was applied compression, if any, once a `VkImage` has been created, add a [VkImageCompressionPropertiesEXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageCompressionPropertiesEXT.html) structure to the `pNext` chain of [VkImageSubresource2EXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageSubresource2EXT.html), and call [vkGetImageSubresourceLayout2EXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetImageSubresourceLayout2EXT.html):
```

Comment:
This sentence needs the second compression removed for a start, but then it's still too long and complicated, needs splitting in 2:

```suggestion
To query which compression was applied, if any, once a `VkImage` has been created, you need to add a [VkImageCompressionPropertiesEXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageCompressionPropertiesEXT.html) structure. VkImageCompressionPropertiesEXT is added to the `pNext` chain of [VkImageSubresource2EXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkImageSubresource2EXT.html), and then you call [vkGetImageSubresourceLayout2EXT](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkGetImageSubresourceLayout2EXT.html):
```